### ISSUE TITLE
Uncaught Error: EBUSY: resource busy or locked

### ISSUE THREAD

**Issue by charlag** [tutao/tutanota#2113]
```
Client: win32
Tutanota version: 3.74.2
User agent:
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) tutanota-desktop/3.74.2 Chrome/80.0.3987.163 Electron/8.2.3 Safari/537.36
Error
Error message: EBUSY: resource busy or locked, open 'C:\Users\MCELIN~1\AppData\Local\Temp\tuta\mLeasingUmowy _ Portal Klienta.pdf'
Stacktrace:
Error: EBUSY: resource busy or locked, open 'C:\Users\MCELIN~1\AppData\Local\Temp\tuta\mLeasingUmowy _ Portal Klienta.pdf'
```

**Comment by charlag** [comment]
Check opening files. Hard to reproduce because file was opened before it was fully downloaded.

**Comment by ganthern** [comment]
Still occurs in 3.76.6:

```
Client: win32
Type: FREE
Tutanota version: 3.76.6
Timestamp (UTC): Thu, 13 Aug 2020 11:04:28 GMT
User agent:
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) tutanota-desktop/3.76.6 Chrome/83.0.4103.119 Electron/9.0.5 Safari/537.36
Error
Error message: EBUSY: resource busy or locked, open 'C:\Users\Windows\AppData\Local\Temp\tuta\ONLINE-VVE-MISE-2018.pdf'
Stacktrace:
Error: EBUSY: resource busy or locked, open 'C:\Users\Windows\AppData\Local\Temp\tuta\ONLINE-VVE-MISE-2018.pdf'
```

**Comment by charlag** [comment]
The original error is gone

**Comment by ganthern** [comment]
Pretty sure this can still happens/can happen again if you "Open" an attachment in a tutanota window, open a second window and try to do the same again. 

Must first fix #4946 and may need blobs though.

**Comment by charlag** [comment]
Seems to be fixed
